build-libs.sh

```diff
@@ -4,26 +4,26 @@ set -e -x
 # Custom path to the node installation in the essentia-emscripten docker image
 # provided along with emsdk. This can be removed once the 
 # entrypoint issues in the essentia-emscripten docker image got resolved.
-NPM_PATH=/emsdk/node/12.18.1_64bit/bin/npm
-NODE_PATH=/emsdk/node/12.18.1_64bit/bin/node
+# NPM_PATH=/emsdk/node/12.18.1_64bit/bin/npm
+# NODE_PATH=/emsdk/node/12.18.1_64bit/bin/node
```

Comment:
yes, I didn't find any issues with the latest emscripten base docker image

Reply:
However I can see the CI build workflow failing because it grabs the pre-built docker image from Docker Hub, so the change to the newer emscripten version is not reflected on the CI workflow. 
I wonder if there is a way to build the image from the Dockerfile on the github CI workflow for these changes to the build pipeline instead of grabbing from Docker Hub...

Reply:
It's definitely possible to build the Docker image in a GitHub workflow, but I should create a new build-dev workflow for when we commit changes to the Dockerfile itself and the other build scripts. 

I can do that on this PR to ensure that the other changes result in a successful build process.

Reply:
I have done this now and checks are passing. I would remove the files being watched on the new "build-dev" workflow from the other two workflows (integration.yml and deploy.yml), let me know what you think about that @albincorreya

Reply:
Great job! We could remove these comments in build-libs.sh since it is not needed anymore?